presto-main/src/main/java/io/prestosql/sql/analyzer/StatementAnalyzer.java

```diff
@@ -21,6 +21,7 @@
 import com.google.common.collect.ImmutableSet;
```

Comment:
How did you notice this?

Reply:
We use a custom authentication / data selection scheme that aligns with our tenant selection logic and uses system properties to move the information around (that is the "one patch we maintain" at Zuora). And it turns out that after creating a view, when executing `select * from <view>`, it reported "not authorized". And lo and behold, this is because at this very place, the system properties do not get moved over.